_layouts/default.html

```diff
@@ -3,7 +3,7 @@
 
   {% include head.html %}
 
-  <body>
+  <body class="dark-theme">
```

Comment:
Do you really need this? If you remove the `.dark-theme` selector from `_sass/_site-footer.scss`, then you won't need this class on the body.